src/infra/core/src/repos/dataset_repository_local_fs.rs

```diff
@@ -611,6 +679,16 @@ impl DatasetStorageStrategy for DatasetSingleTenantStorageStrategy {
         std::fs::rename(old_dataset_path, new_dataset_path).int_err()?;
         Ok(())
     }
+
+    fn resolve_account_dir(
+        &self,
+        _account_name: &AccountName,
+    ) -> Result<(PathBuf, AccountName), ResolveDatasetError> {
+        Ok((
+            self.root.join(DEFAULT_ACCOUNT_NAME),
+            AccountName::new_unchecked(DEFAULT_ACCOUNT_NAME),
+        ))
+    }
```

Comment:
There are no account directories in single-tenant mode, all datasets are stored by name. I have no idea where this path would point to. I assume we should not change directory layout in this ticket. The only place where it is used in the main repository is the creation of datasets:

```
            let (_, canonical_account_name) = self
                .storage_strategy
                .resolve_account_dir(account_name)
                .int_err()?;
```

and it does no even use the path. Can't we just create a dedicated operation to resolve account name only? The full form is only used by multi-tenant strategy internally, which does not require to declare this operation as a part of storage strategy trait.

Reply:
It also usend in multitenant `resolve_dataset_alias` method. I'm not sure I can somehow split it

Reply:
Managed to fix